### ISSUE TITLE
Report HTML alerts in Braille

### ISSUE THREAD

**Issue by vincent-lg** [nvaccess/nvda#9352]
Alerts in HTML, using the "alert" ARIA role, are not reported in Braille.

### Steps to reproduce:

Here's a single HTTML/JavaScript script:

```
<button id="alert-trigger">
  Trigger Alert
</button>
<div id="example" role="alert"></div>

<script>
window.addEventListener('load', function () {

  var button = document.getElementById('alert-trigger');

  button.addEventListener('click', addAlert);

});

/*
* @function addAlert
*
* @desc Adds an alert to the page
*
* @param   {Object}  event  -  Standard W3C event object
*
*/

function addAlert (event) {

  var example = document.getElementById('example');
  var template = document.getElementById('alert-template').innerHTML;

  example.innerHTML = template;

}
</script>
```

This example comes from [the ARIA official examples](https://www.w3.org/TR/wai-aria-practices-1.1/).

### Actual behavior:

Currently, the text is spoken on alert, but nothing is displayed on the Braille display.  Normal?  Perhaps such a feature would flood Braille users with information.

### Expected behavior:

I am one of the NVDA users who doesn't use speech in NVDA, just Braille, so this actually makes information harder to find especially if they are "flash" notifications.  I suggest instead that they could appear in a Braille flash message (gone 5 seconds later or the user could just press a routing button to dismiss the notification, as is already the case).

If this feature actually means drowning the user with information, I suggest to make it an optional feature with either a global setting, or an ability to change this setting based on the website.  That will add complexity for sure, but I'm not entirely sure why NVDA doesn't already present Braille users with this information.

**Comment by Adriani90** [comment]
cc: @leonardder, @Andre9642

**Comment by kayatli** [comment]
I would like this to be fixed as well as I occasionally use NVDA with a braille display with the speech turned off. Professionally, I do accessibility testing and see this having a severe negative impact for Deafblind users.

**Comment by zersiax** [comment]
As I already pointed out in #7756 I think having a toggle to read these can be quite the productivity booster.
There are times when working with braille only is preferred or even the only option, and while I respect that incorrect alert spam can be debilitating in such a scenario, having the option is definitely a requirement for making sure no information is missed in such a scenario.

**Comment by fcnjd** [comment]
Yes, as another user who only uses Braille display, I can only agree to the previous comments and say that productivity could grow a lot if those alerts could be shown on the Braille display as well. In today's world with so many web-apps, those alerts fullfill my personal definition of normal notifications, and therefore should get the possibility to be treated the same, as I see it. I'd be very happy if you could fix that.

**Comment by Adriani90** [comment]
in NVDA advanced settings under the group braille, there is a setting called "report live regions" which is enabled by default. Does this setting not fix this problem?

**Comment by vincent-lg** [comment]
Yes, it seems to allow it. Thanks,
